src/pages/terms-of-use/index.tsx

```diff
@@ -215,6 +216,78 @@ const TermsOfUse = () => {
+                  <td>Submissions to the first Global Stocktake by parties and non-Party stakeholders </td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/NDCREG">NDC Registry</ExternalLink>
+                  </td>
+                  <td>Nationally Determined Contributions</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/ACR">Adaptation Communications Registry</ExternalLink>
+                  </td>
+                  <td>Adaptation Communications</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://napcentral.org/submitted-naps">NAP Central Portal</ExternalLink>
+                  </td>
+                  <td>National Adaptation Plans</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/ttclear/tna/reports.html">TNA Portal</ExternalLink>
+                  </td>
+                  <td>Technology Needs Assessments</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/climatefinance?submissions">Fast-Start Finance Country Reports</ExternalLink>
+                  </td>
+                  <td>Country Finance reports</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://www.ipcc.ch/reports/">IPCC Website</ExternalLink>
+                  </td>
+                  <td>IPCC reports from 2015</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://www.ipcc.ch/copyright/">View</ExternalLink>
```

Comment:
This link refers to "This website" - I don't think this statement protects their information on our website @henry-cpr

Reply:
Raised https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites

Reply:
It does as it is the license under which we are republishing information from their website, so happy to leave for the moment. Agree not all the links are relevant though, I will need to explore with policy team so accepting for now and then we rework -- as we have the unfccc terms first which covers the public domain license for all the unfccc documents I'm happy we're covered